README.md

```diff
@@ -268,3 +266,53 @@ external function.
 Additionally, you'll want to use the [checkout
 action](https://github.com/actions/checkout) to make sure your script file is
 available.
+
+### Use npm packages
+
+Like importing your own files above, you can also use installed modules:
+
+```yaml
+on: push
+
+jobs:
+  echo-input:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm ci
+      # or one-off:
+      - run: npm install -g execa
```

Comment:
Do we need the `-g` option?

Reply:
I tend to use `-g` when installing one-off packages in a job, so that it doesn't modify your `package.json` or lockfile ... but now that I think of it, this wouldn't even work in this case since it wouldn't put the installed module into the local `${process.env.GITHUB_WORKSPACE}/node_modules` folder 🤔

```suggestion
      - run: npm install execa
```